stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -469,9 +469,13 @@ private void remoteOrClusterRead() throws Exception {
     private boolean processFile(String filePath, boolean isRead) throws IOException {
       mCurrentOffset = 0;
       while (!Thread.currentThread().isInterrupted()) {
-        if (isRead && CommonUtils.getCurrentMs() > mContext.getEndMs()) {
-          closeInStream();
-          return true;
+        if (CommonUtils.getCurrentMs() > mContext.getEndMs()) {
+          mFuseIOTaskResult.setEndMs(CommonUtils.getCurrentMs());
```

Comment:
For writing throughput test, the `FuseIOTaskResult.endMs` would still be updated after the `duration` time, and the final `endMs` depends on this value (see [here](https://github.com/Alluxio/alluxio/blob/e0798117d16134f47d49ed764f58da0ec3940df0/stress/common/src/main/java/alluxio/stress/fuse/FuseIOTaskResult.java#L88))and thus would be incorrect.

Reply:
Not quite understand here. 
Here, i want write to stop recording after it pass the context.getEndMs.
so set the end ms here, why it would be incorrect?
I can grab you later to discuss directly